packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.types.ts

```diff
@@ -54,9 +54,13 @@ export interface ISuggestionsProps<T> extends React.Props<any> {
    */
   createGenericItem?: () => void;
   /**
-   * The CSS classname of the suggestions list.
+   * The CSS classname of the suggestions root.
    */
   className?: string;
+  /**
+   * The CSS classname of the suggestionslist
+   */
+  suggestionsClassName?: string;
```

Comment:
>suggestionsClassName [](start = 2, length = 20)

Kind of an anti-pattern by this point in time with the direction most other Fabric component is moving (towards `styled`/merge-styles)...

Reply:
This has been expected to be there, and just not used.

Reply:
Right, I understand that reasoning for existing hooks but to be adding more is kind of moving in the opposite direction..

Reply:
No this isn't adding more, my point is that this is an existing hook. It's already given as an option in the IBasePickersuggetsionsProps, it just never got anywhere.